### PR TITLE
fix description of REVIVER_SEED to contain direct-hit detail

### PR DESCRIPTION
**Pull request by flx-sta**
## What are the changes?

Reviver seed is missing description info about the `direct-hit only` condition

### Translations

- [x] English
- [x] German
- [x] French
- [x] Brazil
- [x] Italian (not 100% checked)
- [x] Spanish
- [x] Korean
- [x] Chinese 1
- [x] Chinese 2

## Why am I doing these changes?

It's missing

## What did change?

added direct hit info for english & german

### Screenshots/Videos

image
image
image
image
image
image
image

## How to test the changes?

It's localziation

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/modifier-type.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "Cendre Sacrée" },
 
-    "REVIVER_SEED": { name: "Résugraine", description: "Réanime et restaure la moitié des PV de son porteur s’il tombe K.O. ." },
+    "REVIVER_SEED": { name: "Résugraine", description: "Réanime et restaure la moitié des PV de son porteur s’il tombe K.O. ." },  //TODO: missing info: after a direct hit only!
```
Comment (Adri1) [inline src/locales/fr/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "Résugraine", description: "Réanime et restaure la moitié des PV de son porteur s’il tombe K.O. d’une capacité directe." },
```

**Review by Adri1 — COMMENTED** [review]
Added French

File: src/locales/pt_BR/modifier-type.ts [hunk 2]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "Cinza Sagrada" },
 
-    "REVIVER_SEED": { name: "Semente Reanimadora", description: "Após desmaiar, reanima com 50% de PS." },
+    "REVIVER_SEED": { name: "Semente Reanimadora", description: "Após desmaiar, reanima com 50% de PS." },  //TODO: missing info: after a direct hit only!
```
Comment (josericardo-fo) [inline src/locales/pt_BR/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "Semente Reanimadora", description: "Após desmaiar por um ataque direto, reanima com 50% de PS." },
```

File: src/locales/zh_CN/modifier-type.ts [hunk 3]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "圣灰" },
 
-    "REVIVER_SEED": { name: "复活种子", description: "恢复1只濒死宝可梦的HP至1/2。" },
+    "REVIVER_SEED": { name: "复活种子", description: "恢复1只濒死宝可梦的HP至1/2。" },  //TODO: missing info: after a direct hit only!
```
Comment (40chyan) [inline src/locales/zh_CN/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "复活种子", description: "受到技能攻击伤害濒死时，\n恢复该宝可梦的HP至1/2。" },  
```

File: src/locales/zh_TW/modifier-type.ts [hunk 4]
```diff
@@ -153,7 +153,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
     SACRED_ASH: { name: "聖灰" },
     REVIVER_SEED: {
       name: "復活種子",
-      description: "恢復1隻瀕死寶可夢的HP至1/2。",
+      description: "恢復1隻瀕死寶可夢的HP至1/2。",  //TODO: missing info: after a direct hit only!
```
Comment (40chyan) [inline src/locales/zh_TW/modifier-type.ts:156]:
```suggestion
      description: "受到技能攻擊傷害瀕死時，\n恢復該瀕死寶可夢的HP至1/2。",

**Review by 40chyan — COMMENTED** [review]
zh_cn & tw translation

**Review by OrangeRed — APPROVED** [review]

File: src/locales/de/modifier-type.ts [hunk 5]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "Zauberasche" },
 
-    "REVIVER_SEED": { name: "Belebersamen", description: "Belebt den Träger mit der Hälfte seiner KP wieder sollte er kampfunfähig werden." },
+    "REVIVER_SEED": { name: "Belebersamen", description: "Belebt den Träger nach einem direkten Treffer mit der Hälfte seiner KP wieder sollte er kampfunfähig werden." },
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "Belebersamen", description: "Belebt den Träger mit der Hälfte seiner KP wieder sollte er  durch einen direkten Treffer kampfunfähig werden." },
```

File: src/locales/de/modifier-type.ts [hunk 6]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "Zauberasche" },
 
-    "REVIVER_SEED": { name: "Belebersamen", description: "Belebt den Träger mit der Hälfte seiner KP wieder sollte er kampfunfähig werden." },
+    "REVIVER_SEED": { name: "Belebersamen", description: "Belebt den Träger mit der Hälfte seiner KP wieder sollte er  durch einen direkten Treffer kampfunfähig werden." },
```
Comment (flx-sta) [inline src/locales/de/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "Belebersamen", description: "Belebt den Träger mit der Hälfte seiner KP wieder,  sollte er  durch einen direkten Treffer kampfunfähig werden." },
```

File: src/locales/es/modifier-type.ts [hunk 7]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "Cen. Sagrada" },
 
-    "REVIVER_SEED": { name: "Semilla Revivir", description: "Revive al portador con la mitad de sus PS al debilitarse." },
+    "REVIVER_SEED": { name: "Semilla Revivir", description: "Revive al portador con la mitad de sus PS al debilitarse." }, //TODO: missing info: after a direct hit only!
```
Comment (ReneGV) [inline src/locales/es/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "Semilla Revivir", description: "Revive al portador con la mitad de sus PS al debilitarse por un golpe directo." },
```

File: src/locales/ko/modifier-type.ts [hunk 8]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "성스러운분말" },
 
-    "REVIVER_SEED": { name: "부활의씨앗", description: "포켓몬이 쓰러지려 할 때 HP를 절반 회복한다." },
+    "REVIVER_SEED": { name: "부활의씨앗", description: "포켓몬이 쓰러지려 할 때 HP를 절반 회복한다." },  //TODO: missing info: after a direct hit only!
```
Comment (EnochG1) [inline src/locales/ko/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "부활의씨앗", description: "포켓몬이 공격을 받고 쓰러지려 할 때 HP를 절반 회복한다." }, 
```

**Review by InnocentGameDev — APPROVED** [review]
Spanish translation is correct

**Review by EnochG1 — APPROVED** [review]

File: src/locales/it/modifier-type.ts [hunk 9]
```diff
@@ -151,7 +151,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SACRED_ASH": { name: "Cenere Magica" },
 
-    "REVIVER_SEED": { name: "Revitalseme", description: "Il possessore recupera 1/2 di PS in caso di svenimento." },
+    "REVIVER_SEED": { name: "Revitalseme", description: "Il possessore recupera 1/2 di PS in caso di svenimento." },  //TODO: missing info: after a direct hit only!
```
Comment (flx-sta) [inline src/locales/it/modifier-type.ts:154]:
```suggestion
    "REVIVER_SEED": { name: "Revitalseme", description: "Il possessore recupera 1/2 di PS in caso di svenimento causato da un colpo diretto." },
```

**Review by Adri1 — APPROVED** [review]

**Review by f-fsantos — APPROVED** [review]

**Pull request merged**